src/DAT/DAT.jl

```diff
@@ -708,7 +708,7 @@ function runLoop(dc::DATConfig, showprog)
             "YAXArrays is not loaded on all workers. Please run `@everywhere using YAXArrays` to fix.",
         )
         mapfun = showprog ? progress_pmap : pmap
-        mapfun(CachingPool(workers()),allRanges, on_error=identity) do r
```

Comment:
I was just thinking that maybe we should pass this as a kwarg to `mapfun` via `dc` with the default as your fix, and then if for whatever reason this is needed in other use cases, then users could potentially set this to `identity`.  That way, we could still support both scenarios.

Reply:
Yeah, but if we actually don't need it, I don't want to add it just in case. If @meggart also doesn't need it I would rather drop it for now and introduce the kwarg option if there is an actual use case.